chispa/default_formats.py

```diff
@@ -1,9 +0,0 @@
-from dataclasses import dataclass
-
-
-@dataclass
-class DefaultFormats:
```

Comment:
For me it looks like a breaking change. What do you think about leaving it as is, just add a deprecation warning and a request to use `FormattingConfig` instead?

Reply:
Good idea, I will implement that!